wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSenderMessageDetailsCell.swift

```diff
@@ -52,6 +52,8 @@ final class ConversationSenderMessageDetailsCell: UIView, ConversationMessageCel
     weak var actionController: ConversationMessageActionController?
 
     private var trailingDateLabelConstraint: NSLayoutConstraint?
+    private(set) var avatarBottomAnchorConstraint: NSLayoutConstraint?
+    private(set) var avatarGreaterThanBottomAnchorConstraint: NSLayoutConstraint?
```

Comment:
Changes in this file fix a constraint issue for deleted messages:
<img width="446" alt="Screenshot 2025-03-28 at 12 19 24" src="https://github.com/user-attachments/assets/75492257-d231-4ed0-a270-579165c787b7" />